compiler/plugins/target/AMD-AIE/aie/AMDAIECreatePathFindFlows.cpp

```diff
@@ -1320,36 +529,36 @@ bool AMDAIEPathfinderPass::attemptFixupMemTileRouting(
     const OpBuilder &builder, SwitchboxOp northSwOp, SwitchboxOp southSwOp,
     ConnectOp &problemConnect) {
   int problemNorthChannel;
-  if (problemConnect.getSourceBundle() == WireBundle::North) {
+  if (problemConnect.getSourceBundle() == StrmSwPortType::NORTH)
     problemNorthChannel = problemConnect.getSourceChannel();
-  } else if (problemConnect.getDestBundle() == WireBundle::North) {
+  else if (problemConnect.getDestBundle() == StrmSwPortType::NORTH)
     problemNorthChannel = problemConnect.getDestChannel();
-  } else
+  else
     return false;  // Problem is not about n-s routing
   int problemSouthChannel;
-  if (problemConnect.getSourceBundle() == WireBundle::South) {
+  if (problemConnect.getSourceBundle() == StrmSwPortType::SOUTH)
     problemSouthChannel = problemConnect.getSourceChannel();
-  } else if (problemConnect.getDestBundle() == WireBundle::South) {
+  else if (problemConnect.getDestBundle() == StrmSwPortType::SOUTH)
     problemSouthChannel = problemConnect.getDestChannel();
-  } else
+  else
```

Comment:
Style guide says to keep braces: https://google.github.io/styleguide/cppguide.html#Formatting_Looping_Branching

Reply:
I guess that style guide says always keep braces? MLIR's convention is to drop them for single line conditonals and loops. If you really want I'll change it back but I think keeping things from being stretched out is beneficial (very small benefit but nonetheless).

Reply:
> I guess that style guide says always keep braces? 

Nope, it doesn't. 

```
// OK - fits on one line.
if (x == kFoo) { return new Foo(); }

// OK - braces are optional in this case.
if (x == kFoo) return new Foo();

// OK - condition fits on one line, body fits on another.
if (x == kBar)
  Bar(arg1, arg2, arg3);

// Bad - `if ... else` statement is missing braces.
if (x) DoThis();
else DoThat();

// Bad - `do ... while` statement is missing braces.
do DoThis();
while (x);
```

IREE follows this style guide, so I think we should be consistent on this.

>  I think keeping things from being stretched out is beneficial (very small benefit but nonetheless).

In the case of  of if/else there's no stretching out by adding braces?

Reply:
> This exception does not apply to multi-keyword statements like if ... else or do ... while.

So basically only single `if`s right?